packages/io/dxf-serializer/tests/geom2ToDxf.test.js

```diff
@@ -46,7 +46,7 @@ test('2D GEOMETRY to DXF POLYLINE', (t) => {
 })
 
 const empty = `999
-DXF generated by JSCAD
+Created by JSCAD
```

Comment:
Lot's of repeat here.  Worth pulling this out into a constant, maybe?

Reply:
Yeah. Lots of different implementations, so lots of duplication.
The problem is where to put the constant. There is only io/io
Suggestions?

Reply:
Although, a common value is possible. I'm opting for no change.
The new phrase is shorter! 👍